tools/lint_list.py

```diff
@@ -13,13 +13,16 @@
 from datetime import timedelta
 
 
-VALID_ENTRY_KEYS = {
+REQUIRED_ENTRY_KEYS = {
```

Comment:
I split these out because otherwise every single field has to have a duplicate `logo: None` that will never be used. It doesn't really matter though. If we do want to keep it we should consider what other keys can be moved, but I was trying to keep this patch as minimal as possible so I didn't include them here.